spec/controllers/exports_controller_spec.rb

```diff
@@ -130,4 +130,30 @@
       expect(response).to render_template(:export_in_progress)
     end
   end
+
+  describe "#spending_breakdown_download" do
+    let(:user) { create(:beis_user) }
+    let(:fake_downloader) { double(:downloader, download: "file contents") }
```

Comment:
Does it need `fake_` in the name? We would have a lot of fakes if all doubles had it!